src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/DataBuilder.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.dluhc.printapi.testsupport.testdata
+
+import java.util.UUID
+
+fun aBatchId() = UUID.randomUUID().toString().replace("-", "")
```

Comment:
I'm not sure if all these should be renamed `aValidBatchId` - seems to be just a bit of noise word - but happy to add 😄

Reply:
@Neil-Massey , for consistency, I think it will be good to have the convention `aValid..`.